api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java

```diff
@@ -14,7 +14,7 @@
     /** TODO: implement dynamic switching of schemas **/
     @Query(value = "select * from cdr.criteria c " +
             "where c.type = :type " +
-            "and match(c.name) against(:value in boolean mode) " +
+            "and (match(c.name) against(:value in boolean mode) or match(c.code) against(:value in boolean mode))" +
```

Comment:
Can you explain why we want to match against the code as well as the name? Are both visible to the user?

Reply:
<img width="562" alt="screen shot 2017-11-15 at 2 18 29 pm" src="https://user-images.githubusercontent.com/30186906/32858109-efe89100-ca0f-11e7-8fba-1e25d1ea1f64.png">
So in the above tree, code is bolded and the text on the right is the name database field

Reply:
So i think the users would find it useful to search by code or something in the name like 'Diabetes'.. which has a bunch of different codes

Reply:
@danqrodney  Think of searching for a book by dewey decimal code or by title.

Reply:
lol.. well put

Reply:
Got it. Thanks!